watcher/watcher.go

```diff
@@ -447,12 +443,11 @@ func (w *Watcher) ReportRoomsStatuses() error {
 // WithLock is a helper function that runs a block of code
 // that needs to hold a type of lock to redis
 func (w *Watcher) WithLock(l *logrus.Entry, lockKey string, f func() error) (lockErr, err error) {
-	lock, _, err := controller.AcquireLock(
+	lock, err := controller.AcquireLockOnce(
```

Comment:
What was the previous behaviour? What is this commit fixing by changing AcquireLock to AcquireLockOnce?

Reply:
AcquireLock use ticker to retry acquiring lock. AcquireLockOnce does not retry.

As watchRooms already runs periodically with ticker, I think it is not necessary to have another ticker to acquire lock.

Also, workers were getting stuck trying to get lock until lockTimeout

Reply:
Okay, makes sense!